Description: use fs-extra's cp instead of using another trivial package we don't have
Author: Julien Puydt
Forwarded: yes

--- a/index.js
+++ b/index.js
@@ -16,7 +16,7 @@
 parsers.parseNames = require('./scripts/parse-names.js');
 parsers.parseNameAliases = require('./scripts/parse-name-aliases.js');
 const extend = utils.extend;
-const cp = require('cp');
+const copySync = require('fs-extra').copySync;
 const jsesc = require('jsesc');
 const template = require('lodash.template');
 
@@ -183,7 +183,7 @@
 		'decode-property-map.js',
 		'decode-ranges.js',
 	].forEach(function(file) {
-		cp.sync(
+		copySync(
 			path.resolve(staticPath, file),
 			path.resolve(__dirname, `output/unicode-${version}/${file}`)
 		);
